src/gui/scene/Renderer.js

```diff
@@ -25,19 +113,31 @@ class Renderer {
 
     /** @type {THREE.Raycaster} */
     #raycaster
-    
+
+    /** @type {GlobalScale} */
+    #globalScale
+
     /**
      * 
      * @param {HTMLCanvasElement} domElement 
      */
     constructor(domElement) {
-        this.#renderer = new THREE.WebGLRenderer({canvas: domElement, alpha: true, antialias: false});
+        this.#renderer = new THREE.WebGLRenderer({canvas: domElement, alpha: true, antialias: true});
```

Comment:
Antialiasing should not have a big impact on the code, if needed we can disable this setting for non webXR environments but for sake of simplicity, I turned it on for all versions.